system/jlib/jptree.cpp

```diff
@@ -1083,8 +1087,10 @@ void PTree::resolveParentChild(const char *xpath, IPropertyTree *&parent, IPrope
             }
             if (!parent)
             {
+#ifdef _DEBUG
```

Comment:
@ghalliday - as the compile failure (in Release) highlights, relating to 'multiplePaths' not used,
the check on 1084-1085 can also be avoided (wrapped in a _DEBUG), along with the declaration.